scripts/perftune.py

```diff
@@ -487,11 +487,11 @@ def tune(self):
             self.__setup_bonding_iface()
 
         # Increase the socket listen() backlog
-        fwriteln_and_log('/proc/sys/net/core/somaxconn', '4096')
+        fwriteln_and_log('/proc/sys/net/core/somaxconn', '16384')
```

Comment:
This change is not related to MLX cards - please, put it a separate patch with a corresponding description.